dockerfiles/build.include

```diff
@@ -0,0 +1,151 @@
+#!/bin/sh
```

Comment:
We don't have to fork :) but it was simpler at the stage where I am. Maybe we can download/curl it ? or do you have any other suggestion ?